components/rootless_studio/README.md

```diff
@@ -0,0 +1,25 @@
+# Rootless Docker Studio
+
+## Building
+
+`./build-docker-image.sh`
+
+### Variables
+
+* HAB_BLDR_CHANNEL (default: "stable")
+* IMAGE_NAME (default: "habitat:default-studio")
+* STUDIO_TYPE (default: "default")
+
+## Running
+
+`docker run -it --rm -v $PWD:/src -v ~/.hab/cache:/hab/cache -e HAB_ORIGIN=$HAB_ORIGIN habitat:default-studio enter`
+
+OR
+
+`docker run -it --rm -v $PWD:/src -v ~/.hab/cache:/hab/cache -e HAB_ORIGIN=$HAB_ORIGIN habitat:default-studio build <optional_path>`
```

Comment:
It might be worth extracting this heinous command into a `Makefile` or shell script or what have you, but not necessary for this particular PR.

Reply:
This is in a script with this PR already that'll get added to buildkite